src/GraphQL/Types/Schema.cs

```diff
@@ -89,7 +90,20 @@ public void Initialize()
         {
             CheckDisposed();
 
-            FindType("____");
+            if (Initialized)
+                return;
+
+            lock (_allTypesInitializationLock)
+            {
+                if (Initialized)
+                    return;
+
+                CreateSchemaTypes();
+
+                Initialized = true;
+
+                FindType("____");
```

Comment:
Can we remove this line?  I think it was only there to trigger `Lazy`, and is not needed now.

Reply:
All tests pass without this line of code.  I'll remove it.

Reply:
Yes, it was here only to trigger Lazy.